var/spack/repos/builtin/packages/nwchem/package.py

```diff
@@ -42,7 +37,7 @@ class Nwchem(Package):
     depends_on("mpi")
     depends_on("scalapack")
     depends_on("fftw-api")
-    depends_on("python@3:3.9", type=("build", "link", "run"), when="@:7.0.2")
+    depends_on("python@3", type=("build", "link", "run"))
```

Comment:
Does v7.0.2 work with Python 3.10 and higher?

Reply:
Yes, it does. I have just verify it.